mysite/account/views.py

```diff
@@ -413,7 +413,7 @@ def invite_someone_do(request):
 # modified trivially in the POST handler.
 
 
-@django_authopenid.views.not_authenticated
+@authenticated
```

Comment:
@willingc that's what was causing the problem. There were some issues with `@django_authopenid.views.authenticated` that's why we were getting open redirect. It had to be replaced with new decorator